recipe/meta.yaml

```diff
@@ -9,21 +9,23 @@ source:
   sha256: 6461f93a842c649922c2c9a9bc9d9c4834110b89de8c4af196a791ab8f42ba3b
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1000
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   build:
+    - {{ compiler('c') }}
+  host:
     - python
-    - setuptools
+    - pip
     - cython
     - numpy 1.11.*
     - scipy
     - pandas
     - patsy
   run:
     - python
-    - numpy >=1.11
+    - {{ pin_compatible('numpy') }}
```

Comment:
@croth1 we must build with the minimum numpy compatible, 1.11 in this case, and run with any numpy compatible with that (>=1.11 here but the jinja magic will do that automatically.)

Reply:
@ocefpaf, thanks for the correction - I am still a bit outdated on the changes in conda-build 3! 👍